example/actions/expect_environment_text/lib/action.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:fluttium/fluttium.dart';
+
+/// {@template expect_environment_text}
+/// Check if the environment text label is set on the main screen.
+///
+/// This action can be invoked using the short-hand version:
+///
+/// ```yaml
+/// - expectEnvironmentText:
+/// ```
+/// {@endtemplate}
+class ExpectEnvironmentText extends Action {
+  /// {@macro expect_environment_text}
+  const ExpectEnvironmentText();
+
+  /// Called when it executes the action in a flow file.
+  @override
+  Future<bool> execute(Tester tester) async {
+    if (!await ExpectVisible(
+      text: "Environment: (Development|Staging|Production|None){1}",
+    ).execute(tester)) {
```

Comment:
```suggestion
    ).execute(_)) {
```

Reply:
I dont think is correct? The tester is needed to execute the action?

Reply:
Oh yeah I completely misread the code...that's what I get for reviewing late at night 💤